routes/conditions/conditions.controller.js

```diff
@@ -10,10 +11,13 @@ module.exports = (app, route) => {
     .get((req, res) => {
       const data = getSessionData(req)
 
-      if (!data.conditions) {
-        res.redirect('/en/add_condition')
+      if (!data.conditions || !data.conditions.length) {
```

Comment:
should we not explicitly check to see if the length is zero instead of "falsey"? 

Makes more sense to me from a code legibility point of view.

Reply:
Will add this explicit test to PR #69